### ISSUE TITLE
CodeMirror 6

### ISSUE THREAD

**Issue by trusktr** [wylieconlon/lsp-editor-adapter#52]
Will this work with CodeMirror 6? I'm not familiar with how similar or different the new CodeMirror API is.

**Comment by marc2332** [comment]
No, it won't, the API is completely different. I would suggest trying -> https://github.com/FurqanSoftware/codemirror-languageserver/ 😄 

(See https://github.com/codemirror/codemirror.next/issues/269)
